### ISSUE TITLE
Date Picker - Contrast ratio low for focused date cells

### ISSUE THREAD

**Issue by ajay1208** [angular/components#18845]
Steps to reproduce:
1. When we use keyboard to access the date picker and try to select the dates.
2. The highlighter does not pass the WCAG color-contrast
 
 
#### Expected Behavior
It should pass the WCAG color-contrast

#### Actual Behavior

It fail WCAG color-contrast
